src/main/kotlin/de/livepoll/api/util/websocket/SubscribeListener.kt

```diff
@@ -24,11 +24,11 @@ class SubscribeListener(
         if (poll == null) {
             throw ResponseStatusException(HttpStatus.NOT_FOUND)
         } else {
+            val url = "/v1/websocket/poll/$slug"
             if (poll.currentItem == null) {
-                throw ResponseStatusException(HttpStatus.CONTINUE)
+                messagingTemplate.convertAndSendToUser(event.user!!.name, url, "{\"id\":${poll.id}}")
```

Comment:
To be honest, I'm not experienced enough with how WebSockets work in Spring to be able to assess this and the simp user stuff that I don't get. Maybe you can explain this to me on Tuesday. But it should be fine since @marcauberer didn't note anything on this.

Reply:
These are the changes that Marc requested. Now he gets the poll id back if there is no live poll item.

Reply:
Thanks @Ordinateur-Hack ;)
I think the code style is okay I think.
I think we should rename the "simp user stuff". Could be [misleading](https://www.urbandictionary.com/define.php?term=Simp) ...